eng/targets/Helix.Common.props

```diff
@@ -44,6 +44,6 @@
     <HelixAvailableTargetQueue Include="(Debian.9.Arm64.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm64v8-a12566d-20190807161036" Platform="Linux" />
   </ItemGroup>
   <ItemGroup Condition="'$(IsWindowsOnlyTest)' != 'true' AND '$(TargetArchitecture)' == 'arm64' AND '$(IsHelixDaily)' == 'true'">
```

Comment:
@halter73 not directly related to this PR but why is `$(IsWindowsOnlyTest)` part of this `Condition`❔

Reply:
I'd guess that is a leftover from before when all the arm queues were non windows only and probably should be removed now

Reply:
I didn't have time to fix all the failing Windows-only aspnetcore-helix-matrix work items when I was on build ops. I fixed the non-Windows-only work item failures with #26840, and I filed #26872 to track removing `$(IsWindowsOnlyTest)` from this condition.

Reply:
That issue is in the Backlog ☹️. Does that mean we'll never get the Windows-only tests working on Windows ARM64❔

Reply:
Backlog doesn't mean closed. It will have to compete with the other issues in the backlog, but I think it's important enough it should make its way back out of the backlog.